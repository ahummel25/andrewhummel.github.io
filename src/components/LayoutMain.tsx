import React, { FC } from "react";
import styled from "@emotion/styled";

const StyledLayoutMain = styled.main`
  display: flex;
  flex-direction: column;
  flex: 1;
`;

interface LayoutMainProps {
  className?: string;
}

const LayoutMain: FC<LayoutMainProps> = ({ children, className }) => (
  <StyledLayoutMain className={className}>{children}</StyledLayoutMain>
);

export default LayoutMain;
