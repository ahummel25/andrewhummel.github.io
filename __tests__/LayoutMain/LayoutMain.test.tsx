import React from 'react';
import renderer from 'react-test-renderer';

import LayoutMain from '../../src/components/LayoutMain';

describe('LayoutMain', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<LayoutMain paddingLeft={190} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
