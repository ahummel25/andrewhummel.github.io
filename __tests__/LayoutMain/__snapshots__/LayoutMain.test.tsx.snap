// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutMain renders correctly 1`] = `
.emotion-0 {
  background: url(test-file-stub);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding-left: 190px;
}

@media (max-width: 992px) {
  .emotion-0 {
    padding-left: 0;
  }
}

.emotion-0 {
  background: url(test-file-stub);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding-left: 190px;
}

@media (max-width: 992px) {
  .emotion-0 {
    padding-left: 0;
  }
}

<main
  className="emotion-0"
/>
`;
